libs/auth/application/src/lib/auth.service.ts

```
@@ -0,0 +1,18 @@
+export class AuthService {
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)